src/Medulla/Kubernetes/Operator/Entities/V1Alpha1ApplicationEntity.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the Medulla Contributors under one or more agreements.
+// The Medulla Contributors licenses this file to you under the Apache 2.0 license.
+// See the LICENSE file in the project root for more information.
+
+using k8s.Models;
+using KubeOps.Operator.Entities;
+using KubeOps.Operator.Entities.Annotations;
+
+namespace ApplicationControllerKubeOps.Controller.Entities;
+
+
+/// <summary>
+/// ApplicationSpec is spec of CRD for application
+/// </summary>
+public struct ApplicationSpec
```

Comment:
Missing XML comments

Reply:
@AndrewBabbitt97  @keithlogan94  how is this missing XML comments? Is it not detailed enough or this file does not contain the desired XML comment? I used V1Alpha1DataEntity.cs as an example.

Reply:
Added XML comments.